src/server/server_family.cc

```diff
@@ -64,11 +77,70 @@ ABSL_FLAG(bool, df_snapshot_format, true,
 ABSL_FLAG(int, epoll_file_threads, 0,
           "thread size for file workers when running in epoll mode, default is hardware concurrent "
           "threads");
+ABSL_FLAG(ReplicaOfFlag, replicaof, ReplicaOfFlag{},
+          "Specifies a host and port which point to a target master "
+          "to replicate. "
+          "Format should be <IPv4>:<PORT> or host:<PORT> or [<IPv6>]:<PORT>");
+ABSL_FLAG(bool, continue_on_replication_fail, false,
```

Comment:
To prevent the process from stopping in case replication via `--replicaof` failed

Reply:
But what's the UX of that? Like, I am explicitly setting the flag because I want my instance to be a `replica` but that fails so it's a now a `master` instance.

Reply:
The default is off, so that's a good UX by default. Perhaps it could be useful in the master is down and one needs to takeover, when the server is started automatically (i.e. not via user in command line)

Reply:
But if that's the case then the `master` would start on a `port` that  was supposed to be used by a `replica`. All clients who thought they connected to a replica are now connected to a master instance? Maybe,  introduce a `fallback_port` flag as well? Again, IMO, I don't really see the value of this flag.  

P.s. I am not pushing this, I am just thinking out loud.